aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -729,48 +781,15 @@ func testAccCheckAWSSpotFleetRequest_IamInstanceProfileArn(
 	}
 }
 
-func TestAccAWSSpotFleetRequest_WithTargetGroups(t *testing.T) {
-	var sfr ec2.SpotFleetRequestConfig
-	rName := acctest.RandString(10)
-	rInt := acctest.RandInt()
-
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckAWSSpotFleetRequestDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccAWSSpotFleetRequestConfigWithTargetGroups(rName, rInt),
-				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckAWSSpotFleetRequestExists(
-						"aws_spot_fleet_request.foo", &sfr),
-					resource.TestCheckResourceAttr(
-						"aws_spot_fleet_request.foo", "spot_request_state", "active"),
-					resource.TestCheckResourceAttr(
-						"aws_spot_fleet_request.foo", "launch_specification.#", "1"),
-					resource.TestCheckResourceAttr(
-						"aws_spot_fleet_request.foo", "target_group_arns.#", "1"),
-				),
-			},
-		},
-	})
-}
-
-func testAccAWSSpotFleetRequestConfigAssociatePublicIpAddress(rName string, rInt int) string {
+func testAccAWSSpotFleetRequestConfigBase(rName string, rInt int) string {
```

Comment:
😄 Thank you. We may want to simplify these test configurations to use the `AWSEC2SpotFleetServiceRolePolicy` managed policy or `AWSServiceRoleForEC2SpotFleet` service linked role in the future.